frontend/public/components/nav/perspective-nav.tsx

```diff
@@ -64,14 +72,15 @@ const PerspectiveNav: React.FC<{}> = () => {
             if (!model) {
               return null;
             }
-            const { labelPlural, apiVersion, apiGroup, namespaced, crd, plural } = model;
+            const { apiVersion, apiGroup, namespaced, crd, plural } = model;
+            const label = getLabelForResource(resource);
             const duplicates =
-              pinnedResources.filter((res) => getLabelForResource(res) === labelPlural).length > 1;
+              pinnedResources.filter((res) => getLabelForResource(res) === label).length > 1;
             const props = {
               key: `pinned-${resource}`,
-              name: labelPlural,
+              name: label,
```

Comment:
Hmmm, so if we can't get a model, but have a model... we need to try and have some default. Can we not use `model.label` as a fallback? We don't ever really want to show an empty sidebar item. Untranslated may not be good, but it's better than nothing.

Reply:
I think he's using the output of the check (including fallback) from line 76 here.

Reply:
Ah, looking closely, I missed the `if (!model)` check... seems odd to fetch it twice but I guess it makes the utility useful for the filter of duplicates.

nvm me! Looks good, just odd flow confused me on first glance haha.